### PR TITLE
Allow feature flags to be passed from install CLI

### PR DESCRIPTION
**Pull request by ashish-amarnath**
Signed-off-by: Ashish Amarnath <ashisham@vmware.com>

### CODE REVIEW

File: changelogs/unreleased/2503-ashish-amarnath [hunk 1]
```diff
@@ -0,0 +1 @@
+allow feature flags to be passed from install CLI #2503
```
Comment (skriss) [inline changelogs/unreleased/2503-ashish-amarnath:1]: you can drop the PR # from the text :)

File: pkg/cmd/cli/install/install.go [hunk 2]
```diff
@@ -101,6 +102,7 @@ func (o *InstallOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.Var(&o.Plugins, "plugins", "Plugin container images to install into the Velero Deployment")
 	flags.BoolVar(&o.CRDsOnly, "crds-only", o.CRDsOnly, "only generate CustomResourceDefinition resources. Useful for updating CRDs for an existing Velero install.")
 	flags.StringVar(&o.CACertFile, "cacert", o.CACertFile, "file containing a certificate bundle to use when verifying TLS connections to the object store. Optional.")
+	flags.StringVar(&o.Features, "features", o.Features, "comma separated list of Velero feature flags to be set on the Velero deployment")
```
Comment (skriss) [inline pkg/cmd/cli/install/install.go:105]: I think we probably want to set these on the restic DaemonSet too, if we're creating it - @nrb WDYT?

File: changelogs/unreleased/2503-ashish-amarnath (hunk 1 above)
Comment (ashish-amarnath) [reply changelogs/unreleased/2503-ashish-amarnath:1]: copy-pasta

File: pkg/cmd/cli/install/install.go (hunk 2 above)
Comment (nrb) [reply pkg/cmd/cli/install/install.go:105]: Yep, I agree.

Comment (ashish-amarnath) [reply pkg/cmd/cli/install/install.go:105]: I can do that. I'll need to confirm that this doesn't trigger the `unbound flags` error

Comment (ashish-amarnath) [reply pkg/cmd/cli/install/install.go:105]: done! Testing also looked good. No `unknown flag` errors

Comment (skriss) [inline pkg/cmd/cli/install/install.go:105]: let's update the flag help text to mention that they're passed to the restic daemonset too (if it's being installed)

**Pull request merged**